### ISSUE TITLE
Add Form 7 and a 2006 FR notice and update form instructions in the Guidance Search

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#5818]
### Summary

**What we're after:**
Form 7 and a 2006 FR notice are supposed to be in the Guidance Search but for whatever reason they are not. We need to add them to the guidance search PDF site map and @johnnyporkchops will need to add it to his S3 drawer for his sitemap work.

Related to Zip code updating, a few form instructions will need to have replacement files uploaded and those time stamps will need to be updated in the site map.

### Related issues

#5782 

### Completion criteria

- [ ] Form 7 (https://www.fec.gov/resources/cms-content/documents/fecform7.pdf) is added to the guidance search site map 
- [x] This FR notice (https://www.fec.gov/resources/cms-content/documents/fedreg_notice_2006-23_EO13892.pdf) is added to site map
- [x] Replacement form instruction files uploaded to Wagtail
- [ ] Site map updated for form instruction files that were uploaded
- [ ] Updated site map xml file is uploaded to Wagtail
- [ ] Updated site map code is placed in GitHub.
- [x] Form 7, the FR notice and the replacement files for form instructions are added to the S3 bucket that will be used for guidance documents (coordinate with @johnnyporkchops) - John plans to grab all things in the bucket right before deployment to capture latest versions.

### Tech steps or considerations (optional)

[How-to-guide](https://docs.google.com/document/d/1hfvlYhGVNF0Km5vrAYtJak2cdAYLYQPbiPUfLxbRN1E/edit#heading=h.83p10qypinz4)

**Comment by dorothyyeager** [comment]
All replacement form instructions except Form 7 are in Wagtail. Site map for PDF updated with the FR notice and the form instructions time stamp. 

We have a question about Form 7 we need to resolve with Info; waiting on their response. Then will finish sitemap update and move PR to ready status to update the code in our repo.

**Comment by dorothyyeager** [comment]
We are done for everything but Form 7 and its instructions. Waiting on Info, who is waiting on EFO. When I get their blessing, will move out of blocked, upload those two files and update guidance search.

**Comment by dorothyyeager** [comment]
We have learned that many forms will be modified as part of the Zip code update project, and will tackle Form 7 and those other forms in a new ticket (https://github.com/fecgov/fec-cms/issues/5846) when they are ready.
